### DISCUSSION TITLE
Asus router wifi disconnect - Temperature ?

### DISCUSSION THREAD

**Discussion by SimonFili** [arendst/Tasmota#19142] [Support request]
Hi all,

I've been struggling with Tasmota disconnect for a long while in one of my HA/Tasmota installation. Then I added some Wi-Fi camera and started to saw the same issue.

After trying so many combinations of router config and Tasmota config, I finally found a correlation between the CPU temperature of the router and the disconnect of ALL my Tasmota devices.

Here's the connection status (grey is unavailable) and the CPU temperature.  Quite revealing! 
![image](https://github.com/arendst/Tasmota/assets/7669537/4021d6e6-66a8-4a80-bc14-921722295ece)

So I'm wondering if all the people who saw those disconnect issue saw/see this?

My next step will be to put a temperature sensor outside the router close to the case and see if there's a correlation.
Then if there's a correlation, either add a fan if possible, but I will also open a ticket with ASUS.

**Comment by Jason2866** [comment]
Mhh, maybe something unusual for the Asus Router in the espressif wifi stack produces a high CPU load and the router gets crazy hot.

**Comment by SimonFili** [comment]
The only thing I see on the Tasmota side is this:
08:01:44.090 WIF: Sending Gratuitous ARP
08:01:55.522 WIF: Checking connection...
08:02:15.538 WIF: Checking connection...
08:02:35.538 WIF: Checking connection...
08:02:44.081 WIF: Sending Gratuitous ARP

Which seems normal and by design.

I have checked the logs of the router, nothing jumps out.

I will open a case with Asus

**Comment by SimonFili** [comment]
Just got a couple of exchanges with ASUS support.  So far, they only pointed to the obvious FAQ and never answered the question about high CPU temp spikes.

**Reply by Jason2866** [reply]
If Asus would be interested to solve the had enough time to do. Imho Asus don't care about issues hitting only a few users. The usual way for consumer electronics. Bug hunting just costs money and "wastes" time.
